lib/cartopy/crs.py

```diff
@@ -2493,6 +2493,33 @@ def transform_points(self, src_crs, x, y, z=None, trap=False):
         return result
 
 
+class WinkelTripel(_WarpedRectangularProjection):
+    """
+    A Winkel-Tripel projection.
+
+    Compromise modified azimuthal projection that is less distorted
+    and more area-accurate. It is comparable to the Robinson
+    projection in both appearance and popularity.
+
+    The National Geographic Society uses the Winkel-Tripel projection
+    for most of the maps they produce.
+    """
+
+    def __init__(self, central_longitude=0.0, globe=None):
+        globe = globe or Globe(semimajor_axis=WGS84_SEMIMAJOR_AXIS)
+        proj4_params = [('proj', 'wintri'),
+                        ('lon_0', central_longitude),
+                        ('lat_0', 0.0)]
```

Comment:
Do we need to pass in the `lat_0` parameter even if 0 is the default, and does that even work, it looks like it is `lat_1` in the documentation: https://proj.org/en/9.4/operations/projections/wintri.html